bin/test_travis.sh

```diff
@@ -34,6 +34,7 @@ if [[ -n "${TEST_OPT_DEPENDENCY}" ]]; then
     python bin/test_external_imports.py
     python bin/test_submodule_imports.py
     python bin/test_executable.py
+    python bin/test_lazy_function.py
```

Comment:
I would rather not add a new test script just for this. We already have too many.

Really it's fine if this function is not fully tested. The main points is supposed to be measuring import time so really we should try to find a way of measuring that and picking up on regressions but that doesn't need to be in this PR.

Travis isn't used any more except for the dev docs build so the `test_travis.sh` script doesn't do anything.

Reply:
@oscarbenjamin @asmeurer I removed the addition to `test_travis.sh`. Shall I

1) Remove the test file altogether

or 

2) Merge the `bin/test_lazy_function.py` with `bin/test_submodule_imports.py` as Aaron suggested

Reply:
There is actually one thing that the `bin/test_travis.sh` script is still used for which is to deploy the dev-docs:
https://github.com/sympy/sympy/blob/64e187e9521cf63653a831c5c3edc956715207ce/.travis.yml#L116-L122
That one job only runs on the master branch (after a PR is merged). We should really replace that with something that runs in GitHub Actions.

> 2\. Merge the `bin/test_lazy_function.py` with `bin/test_submodule_imports.py` as Aaron suggested

That seems reasonable.